src/main/resources/db/migration/V202409231531__create_employee_and_company.sql

```diff
@@ -0,0 +1,15 @@
+-- Create company table
+CREATE TABLE company (
```

Comment:
In your case, this is not a problem that you have migrations and code in the same PR.
But for general knowledge, in bob the pipeline prevents it. We don't want to mix DDLs and code. So in that case you would have had to fix this issue with the rebase